trainer/constants.py

```diff
@@ -30,3 +30,15 @@
     "train": os.path.join(DATA_DIR, "rd-train.tsv"),
     "validation": os.path.join(DATA_DIR, "rd-validation.tsv")
 }
+ML_SEQ_TSV_PATH = {
+    "train": os.path.join(DATA_DIR, "ml-train.tsv"),
+    "validation": os.path.join(DATA_DIR, "ml-validation.tsv")
+}
+ML_TAGS_TSV_PATH = {
+    "train": os.path.join(DATA_DIR, "ml-tags-train.tsv"),
+    "validation": os.path.join(DATA_DIR, "ml-tags-test.tsv")
```

Comment:
nit: consistent naming would be nice, but no need to fix, just fyi.

Reply:
fixed it